frontend/pages/[resourceType].tsx

```diff
@@ -0,0 +1,107 @@
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import { Button, Grid, Divider } from '@mantine/core';
+import Link from 'next/link';
+
+/**
+ * Component which displays list of all resources of some type as passed in by (serverside) props
+ * @returns component with list of resource (by id) buttons that are links to that resource's details
+ */
+export default function ResourceList({ ids, resourceType }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const idItems = ids.map(id => {
+    return (
+      <Link href={`/${resourceType}/${id}`} key={id}>
+        <Button
+          fullWidth
+          color="cyan"
+          radius="md"
+          size="md"
+          variant="subtle"
+          styles={() => ({
+            root: {
+              padding: '2px'
+            },
+            inner: {
+              justifyContent: 'left'
+            }
+          })}
+        >
+          <div>
+            {resourceType}/{id}
+          </div>
+        </Button>
+      </Link>
+    );
+  });
+  return (
```

Comment:
I would add a back button on this page so the user can return to the home page!

Reply:
Added back button and homepage link for top title.